### DISCUSSION TITLE
hao to set the next task or subprocess will be run when this task fail?

### DISCUSSION THREAD

**Discussion by pub1212** [apache/dolphinscheduler#7640] [Q&A]
hao to set the next task or subprocess will be run when this task fail?
case:Some new task relesase to product env,I want to ignore some fail,avoid affecting the main line.

       对于新上线任务，希望无论失败还是成功，都能正常执行下游的任务和流程。这样能够避免新任务上线初期，非预期的bug对主线流程的影响，可以设置策略吗

**Comment by Tandoy** [comment]
As far as I know, there is currently no strategy for your appeal. However, you can try to add a condition node downstream of the new task and set the condition to OR. In this way, the downstream task will be executed regardless of whether the new online task fails or succeeds.

[https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/guide/task/conditions.html](url)
